flexmeasures/ui/templates/crud/asset.html

```diff
@@ -798,37 +981,345 @@ <h5 class="card-title">${sensor.name}</h5>
+        flexSelect.innerHTML = `<option value="blank">Select an option</option>`;
+        for (const [key, value] of Object.entries(assetFlexContextTemplate)) {
+            if (value === null) {
+                flexSelect.innerHTML += `<option value="${key}">${key.split("-").map((word) => word.charAt(0).toUpperCase() + word.slice(1)).join(" ")}</option>`;
+            }
+        }
+
+    }
+
+    function renderFlexInputOptions(contextKey, isArray = false) {
+        return `
+                <div class="card-header">
+                    <ul class="nav nav-tabs card-header-tabs" id="pills-tab" role="tablist">
+                        <li class="nav-item" role="presentation">
+                            <button class="nav-link active bg-fixed-val" id="pills-homfixed" data-bs-toggle="pill" data-bs-target="#pills-fixed" type="button" role="tab" aria-controls="pills-fixed" aria-selected="true">
+                                Fixed Value
+                                <i class="fa fa-info-circle ps-2"
+                                    data-bs-toggle="tooltip"
+                                    data-bs-placement="bottom"
+                                    title="'Enter a fixed value with a unit, e.g. "75kW" or "105 EUR". It will be used in all schedules for this asset and his sensors."
+                                ></i>
+                            </button>
+                        </li>
+                        <li class="nav-item" role="presentation">
+                            <button class="nav-link bg-dynamic-val" id="pills-dynamic-tab" data-bs-toggle="pill" data-bs-target="#pills-dynamic" type="button" role="tab" aria-controls="pills-dynamic" aria-selected="false">
+                                Dynamic Value(a Sensor)
+                                <i class="fa fa-info-circle ps-2"
+                                    data-bs-toggle="tooltip"
+                                    data-bs-placement="bottom"
+                                    title="You can a sensor which provides the values in a dynamic fashion. In the search form below, 
+                                    you can choose the right one. Be sure the data will be available consistently, though, so scheduling will work."
+                                ></i>
+                            </button>
+                            </button>
+                        </li>
+                    </ul>
+
+                    
+                    <div class="tab-content" id="pills-tabContent">
+                        <div class="tab-pane fade show active bg-fixed-val rounded-bottom" id="pills-fixed" role="tabpanel" aria-labelledby="pills-fixed-tab">
+                            <div id="power-capacity-input" class="flex-input-group mb-4 p-2">
+                                <label for="power-capacity" class="form-label">Value</label>
+                                <input type="text" id="power-capacity" class="form-control"
```

Comment:
Please fill this with the existing value if we select a field on the left (which has a value).

E.g. the value is "365 EUR/MWh", and the user wants it to be "366 EUR/MWh", then he only needs to change one number, not retype everything.